File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    answer = ''
    array = []
    s = s.split(" ")
    for i in s:
        array.append(int(i))
    
    min_number = str(min(array))
    answer += ''.join(min_number + " ")
    max_number = str(max(array))
    answer += ''.join(max_number)
    
    return answer
    

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.cpp
#include <string>
#include <vector>
#include <algorithm>
#include <climits>
#include <sstream>
using namespace std;

string solution(string s) {
    stringstream ss(s);
    string number;
    int min_val = INT_MAX;
    int max_val = INT_MIN;
    
    while (ss >> number) {
        int num = stoi(number);
        min_val = min(min_val, num);
        max_val = max(max_val, num);
    }
    
    return to_string(min_val) + " " + to_string(max_val);
}